src/abstractdatatable/abstractdatatable.jl

```diff
@@ -378,6 +378,32 @@ function StatsBase.describe(io, dt::AbstractDataTable)
         println(io, )
     end
 end
+function StatsBase.describe{T <: Real}(io::IO, X::Vector{Union{T, Null}})
+    nullcount = sum(isnull, X)
```

Comment:
I usually prefer `count` for this sort of thing, but `sum` is equivalent so it doesn't really matter